packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/network/ProgressResponseBody.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.modules.network
+
+import okhttp3.MediaType
+import okhttp3.ResponseBody
+import okio.Buffer
+import okio.BufferedSource
+import okio.ForwardingSource
+import okio.Source
+import okio.buffer
+import java.io.IOException
+
+public class ProgressResponseBody public constructor(
+    private val responseBody: ResponseBody,
+    private val progressListener: ProgressListener
+) : ResponseBody() {
+  private lateinit var bufferedSource: BufferedSource
+  private var totalBytesRead = 0L
+
+  public override fun contentType(): MediaType? {
+    return responseBody.contentType()
+  }
+
+  override fun contentLength(): Long {
+    return responseBody.contentLength()
+  }
+
+  public fun totalBytesRead(): Long {
+    return totalBytesRead
+  }
```

Comment:
Let's move those 3 to expression syntax?

```suggestion
  public override fun contentType(): MediaType? = responseBody.contentType()

  override fun contentLength(): Long = responseBody.contentLength()

  public fun totalBytesRead(): Long = totalBytesRead
```